hadoop-hdds/managed-rocksdb/src/main/java/org/apache/hadoop/hdds/utils/db/managed/ManagedRocksDB.java

```diff
@@ -102,4 +111,15 @@ public void deleteFile(LiveFileMetaData fileToBeDeleted)
     File file = new File(fileToBeDeleted.path(), fileToBeDeleted.fileName());
     ManagedRocksObjectUtils.waitForFileDelete(file, Duration.ofSeconds(60));
   }
+
+  public static Map<String, LiveFileMetaData> getLiveMetadataForSSTFiles(RocksDB db) {
+    return db.getLiveFilesMetaData().stream()
+        .filter(liveFileMetaData -> liveFileMetaData.fileName().endsWith(SST_FILE_EXTENSION)).collect(
```

Comment:
`db.getLiveFilesMetaData()` only returns table files, why do we need to filter it?
RocksDB documentation:
```
  /**
   * Gets a list of all table files metadata.
   *
   * @return table files metadata.
   */
  public List<LiveFileMetaData> getLiveFilesMetaData() {
    return Arrays.asList(getLiveFilesMetaData(nativeHandle_));
  }
  ```

Reply:
It could also return manifest and log files

Reply:
removed the filtering logic. I thought we are also returning log file as part of this.